### ISSUE TITLE
Investigate MS-Comet 

### ISSUE THREAD

**Issue by eu9ene** [mozilla/firefox-translations-models#88]
It's a Comet-based model trained on a larger set of human evaluations, so should be more precise.

https://github.com/MicrosoftTranslator/MS-Comet

**Comment by marco-c** [comment]
I had filed this on https://github.com/mozilla/firefox-translations-evaluation/issues/17.

**Comment by eu9ene** [comment]
Yeah, I copied two issues about metrics from there since I archived this repo.

**Comment by eu9ene** [comment]
Right, I see, I didn't know about this one :)
